apps/designer/app/auth/login.tsx

```diff
@@ -44,59 +46,73 @@ export const Login = ({ errorMessage }: { errorMessage: string }) => {
       justify="center"
     >
       <Card size={2}>
-        <Flex direction="column" gap="4" align="center">
+        <Flex direction="column" align="center" gap="3">
           <Heading size="2">Login</Heading>
-          {errorMessage.length ? (
-            <__DEPRECATED__Text css={{ textAlign: "center" }} variant="red">
-              {errorMessage}
-            </__DEPRECATED__Text>
-          ) : null}
-          <Flex gap="2" direction="column" align="center">
-            <Form action="/auth/github" method="post">
-              <LoginButton enabled={data.isGithubEnabled}>
-                <Flex gap="1">
-                  <GithubIcon width="20" />
-                  Login with GitHub
-                </Flex>
-              </LoginButton>
-            </Form>
-            <Form action="/auth/google" method="post">
-              <LoginButton enabled={data.isGoogleEnabled}>
-                <Flex gap="1">
-                  <GoogleIcon width="20" />
-                  Login with Google
-                </Flex>
-              </LoginButton>
-            </Form>
-            {data.isDevLogin && (
-              <>
-                {isDevLoginOpen ? (
-                  <Form action="/auth/dev" method="post">
+          <Flex direction="column" gap="4" css={{ width: 300 }}>
+            {errorMessage.length ? (
+              <__DEPRECATED__Text css={{ textAlign: "center" }} variant="red">
+                {errorMessage}
+              </__DEPRECATED__Text>
+            ) : null}
+            <Flex gap="2" direction="column">
+              <Form action="/auth/github" method="post">
+                <LoginButton enabled={data.isGithubEnabled}>
+                  <Flex gap="2" align="center">
+                    <Flex css={{ width: 20, height: 20 }}>
+                      <GithubIcon />
+                    </Flex>
+                    Login with GitHub
+                  </Flex>
+                </LoginButton>
+              </Form>
+              <Form action="/auth/google" method="post">
+                <LoginButton enabled={data.isGoogleEnabled}>
+                  <Flex gap="2" align="center">
+                    <Flex css={{ width: 20, height: 20 }}>
```

Comment:
additional div to hold the dimensions looks non-ideal, what are some alternatives?

Reply:
also where are those 20px coming from?

Reply:
I added this since it would otherwise collapse the icon as far as I remember. I think the way how icon is made now needs reworking. I'd like Icon to have `size` prop that would define the size internally + currently Icon is not accessible so we probably need add a `label` prop so it can be used by assistive tech. Let's do a follow up for this?

Reply:
sure, please create an issue, to have a todo and so we can do it before we add more workarounds